.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -6,14 +6,22 @@
 
 Binary change:
 <!---
-Please fill in the below table using the size of the Demo app, as found in Finder, from 
-the latest state of the branch you are merging in to and the latest state of your changes.
-In order to get an accurate measurement for iOS, please build the Demo app using the
-Demo.Release scheme for "Any iOS Device (arm64)"
+Please fill in the table below with the binary size of files changed from the latest state 
+of the branch you are merging into and the latest state of your changes. In order to get an 
+accurate measurement of our framework, follow these instructions:
+  1. Change scheme to Demo.Release for Any iOS Device (arm64)
+  2. Build, then navigate to left panel: FluentUI -> Products -> libFluentUI.a
+  3. Show file in Finder, Get Info, record libFluentUI.a binary size.
+
+For individual files:
+  1. Open terminal, "ar x <path of libFluentUI.a>" (no quotes or brackets). This will 
+     generate your individual .o files in your current directory.
```

Comment:
Might be worth moving the "files in your current directory" earlier, or add direction to navigate out of the git repo or something first, so they don't just dump all the .o files in their local repo.